pysim/compositesystem.pyx

```diff
@@ -73,6 +105,17 @@ cdef class CompositeSystem(SimulatableSystem):
         bs_names =  bytes(name,'utf-8')
         bs_desc = bytes(description,'utf-8')
         self._c_sys.add_matrix_inport(bs_names, initialvalue, bs_desc)
+        
+    def expand_single_input(self, portname, sys, sysport, initialvalue):
```

Comment:
This should also work with matrices. Some docstring explaining what "expand" is.

Reply:
How does a "pysim"-matrix look like in python? A nested list or a numpy array?